src/api/address-name-service.js

```diff
@@ -14,11 +14,9 @@ module.exports = {
   async resolveAddress(address) {
     try {
       if (isUndefined(address)) throw TypeError('address is not defined');
-      const { data } = await axios.get(`${getNakaBaseUrl()}/resolve-address`, {
-        params: {
-          address,
-          apikey: CONFIG.API_KEY,
-        },
+      const { data } = await axios.post(`${getNakaBaseUrl()}/resolve-address`, {
```

Comment:
nakabase api schema for post is different, you now need to include api key in the url.

`${getNakaBaseUrl()}/APIKEY/resolve-address`

Remove the apikey from the body